src/algorithms/planargradient.jl

```diff
@@ -0,0 +1,13 @@
+struct PlanarGradient <: NeutralLandscapeMaker
+    direction::Float64
+    PlanarGradient(x::T) where {T <: Real} = new(mod(x, 360.0))
+end
+
+PlanarGradient() = PlanarGradient(360rand())
+
+function landscape(alg::PlanarGradient, s::Tuple{IT,IT}=(10,30)) where {T <: Real, IT <: Integer}
```

Comment:
Why is the size a tuple? Most of the Array constructors, rand etc take two separate args

Reply:
that's mostly shorthand to say "this should have two values", but that's a good point, I can change that